File: c_sharp/Enumeration/Enumeration/EducationStatus.cs
using System.ComponentModel.DataAnnotations;

namespace PeerageHRIS.Enumeration
{
    public enum EducationStatus
    {
        [Display(Name = "In Progress")]
        InProgress,

        [Display(Name = "Completed")]
        Completed, 
        [Display(Name = "On Hold")]
        OnHold,

        [Display(Name = "Fraud")]
        Fraud
    }
}

File: lib/core/enums/education_status.dart
// Represents the current status of an educational qualification.
import 'package:freezed_annotation/freezed_annotation.dart';

part 'education_status.g.dart';

@JsonEnum(alwaysCreate: true)
enum EducationStatus {
  @JsonValue(0)
  inProgress,

  @JsonValue(1)
  completed,

  @JsonValue(2)
  onHold,

  @JsonValue(3)
  fraud,
}